src/ontology/mondo.Makefile

```diff
@@ -638,6 +638,38 @@ release_diff: reports/mondo_release_diff.md
 all: reports/mondo_release_diff.md
 all: reports/mondo_obsoletioncandidates.tsv
 
+##################
+### KGCL Diff ####
+##################
+
+KGCL_ONTOLOGY=mondo-base.obo
+
+prepare_release: kgcl-diff
+
+.PHONY: kgcl-diff
+kgcl-diff: kgcl-diff-release-base
+
+.PHONY: kgcl-diff-release-base
+kgcl-diff-release-base: reports/difference_release_base.yaml \
+						reports/difference_release_base.tsv \
+						reports/difference_release_base.md
+
+tmp/mondo-released.obo:
```

Comment:
```suggestion
tmp/mondo-released.obo: .FORCE
```

I know this is a last resort dependency but we should consider it as this code should always be re-run when the diff is rerun. The SOP takes care of it as well, as it instructs the use of -B, but this is just safer